src/profile/ProfileFooter.js

```diff
@@ -21,8 +21,8 @@ export default function ProfileFooter(props) {
                     </div>
                     <div className="col-md-8">
                         <div className="social align-items-center">
-                            <SocialIcon url="https://twitter.com/jaketrent" />
-                            <SocialIcon url="https://github.com/jaketrent" />
+                            <SocialIcon url="https://twitter.com/jaketrent" style={{ height: 40, width: 40 }}/>

```

Comment:
add a className here and use it in the css instead of inline style

Reply:
Nabs completed dis